README.md

```diff
@@ -0,0 +1,8 @@
+# Pebble build tool
```

Comment:
It's the Pebble Tool, not the Pebble Build Tool.

Reply:
I'll fix that now get it done. If things look good I can squash changes into a single commit.

Thanks for taking the time for review!